src/components/loaders/loader.scss

```diff
@@ -1,5 +1,12 @@
 .loaderContainer {
-  height: calc(100vh - var(--omrs-navbar-height) - var(--workspace-header-height));
+  height: var(--desktop-workspace-window-height);
+}
+
+// tablet
+:global(.omrs-breakpoint-lt-desktop) {
+  .loaderContainer {
+    height: var(--tablet-workspace-window-height);
+  }
```

Comment:
These changes ensure the loading screen takes up the appropriate height based on the viewport.

![CleanShot 2024-05-21 at 11  43 18@2x](https://github.com/openmrs/openmrs-form-engine-lib/assets/8509731/4c8deafa-de02-438c-a44f-49b4b760b3f2)